src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -745,6 +772,13 @@
       },
 
       async sendTransaction(): Promise<void> {
+        this.errors.transfer.amount = false
+        this.errors.transfer.balance = false
+        if (this.amount <= 0) {
+          this.errors.transfer.amount =
+            "Le montant à transférer doit être un nombre positif"
+          return
+        }
```

Comment:
Same here: we are in a `sendTransaction`, I'm not sure these tests should be here. It seems a very small change and handy one to put them at least in a check that would allow on the fly verification and enabling/disabling of the sent button.

Reply:
Let's do that in another PR.